graphql/schemabuilder/output.go

```diff
@@ -125,6 +125,7 @@ func (sb *schemaBuilder) buildStruct(typ reflect.Type) error {
 				return err
```

Comment:
I think it might be cleaner to add setting the description in the functions that build the field (eg in  buildFunctionAndFuncCtx, buildBatchFunctionAndFuncCtx, buildPaginatedFieldWithFallback, and buildFederatedFunction). Also test plans for those cases to make sure the descriptions bubbled up correctly into the introspection schema.